etna/datasets/tsdataset.py

```diff
@@ -760,60 +762,189 @@ def tail(self, n_rows: int = 5) -> pd.DataFrame:
+
+        # combine information
+        segments_dict["end_timestamp"] = [common_dict["end_timestamp"]] * len(segments)
+        segments_dict["num_segments"] = [common_dict["num_segments"]] * len(segments)
+        segments_dict["num_exogs"] = [common_dict["num_exogs"]] * len(segments)
+        segments_dict["num_regressors"] = [common_dict["num_regressors"]] * len(segments)
+        segments_dict["num_regressors"] = [common_dict["num_regressors"]] * len(segments)
+        segments_dict["freq"] = [common_dict["freq"]] * len(segments)
+
+        result_df = pd.DataFrame(segments_dict, index=segments)
+        columns_order = [
+            "start_timestamp",
+            "end_timestamp",
+            "length",
+            "num_missing",
+            "num_segments",
+            "num_exogs",
+            "num_regressors",
+            "freq",
+        ]
+        result_df = result_df[columns_order]
+        result_df.index.name = "segments"
+        return result_df
+
+    def info(self, segments: Optional[Sequence[str]] = None) -> None:
+        """Overview of the dataset that prints the result.
+
+        Method describes dataset in segment-wise fashion.
+
+        Information about dataset in general:
+        * end_timestamp: ending of the segment, missing values are not ignored
+        * num_segments: total number of segments
+        * num_exogs: number of exogenous features
+        * num_regressors: number of exogenous factors, that are regressors
+        * freq: frequency of the dataset
+
+        Information about individual segments:
+        * start_timestamp: beginning of the segment, missing values in the beginning are ignored
+        * length: length according to start_date and end_date
+        * num_missing: number of missing variables between start_date and end_date
+
 
+        Parameters
+        ----------
+        segments:
+            segments to show in overview, if None all segments are shown.
+
+        Examples
+        --------
+        >>> from etna.datasets import generate_const_df
+        >>> df = generate_const_df(
+        ...    periods=30, start_time="2021-06-01",
+        ...    n_segments=2, scale=1
+        ... )
+        >>> df_ts_format = TSDataset.to_dataset(df)
+        >>> regressors_timestamp = pd.date_range(start="2021-06-01", periods=50)
+        >>> df_regressors_1 = pd.DataFrame(
+        ...     {"timestamp": regressors_timestamp, "regressor_1": 1, "segment": "segment_0"}
+        ... )
+        >>> df_regressors_2 = pd.DataFrame(
+        ...     {"timestamp": regressors_timestamp, "regressor_1": 2, "segment": "segment_1"}
+        ... )
+        >>> df_exog = pd.concat([df_regressors_1, df_regressors_2], ignore_index=True)
+        >>> df_exog_ts_format = TSDataset.to_dataset(df_exog)
+        >>> ts = TSDataset(df_ts_format, df_exog=df_exog_ts_format, freq="D")
+        >>> ts.info()
+        <class 'etna.datasets.TSDataset'>
+        end_timestamp: 2021-06-30 00:00:00
```

Comment:
actually `end_timestamp` can be different for different series
I can run smth like 
```
ts = TSDataset(...)
ts.info()
```
and should get valid result